feature/gribi/ate_tests/backup_nhg_multiple_nh_test/backup_nhg_multiple_nh_test.go

```diff
@@ -36,6 +38,8 @@ const (
 	dstPfx        = "203.0.113.0/24"
 	dstPfxMin     = "203.0.113.0"
 	dstPfxMask    = "24"
+	vrf1          = "VRF-A"
```

Comment:
@ xw-g 
Can you be more elaborate , In READE vrfs mentioned are VRF-A and VRF-B

Reply:
I meant that it would be more convenient to name the variable as `vrfA` and `vrfB` so that readers won't have to keep the `vrf1 is VRF-A` and `vrf2 is VRF-B` mapping in their head. :)

Reply:
@agvarghe-cisco agree?

Reply:
Changed to vrfA and vrfB